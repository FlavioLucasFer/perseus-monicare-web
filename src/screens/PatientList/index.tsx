import React, { useEffect } from 'react';
import PatientCard from 'components/PatientCard';

const PatientList = () => {
  return (
    <div className='container' style={{ height: '100vh' }}>
      <div className='row col-sm-12'>
        <PatientCard patient={{
          name: 'teste com nome bem grande',
          age: 20,
          caregiver: 'teste com nome bem grande de novo novamente outra vez',
          status: 'bad',
        }} />

        <PatientCard patient={{
          name: 'teste com nome bem grande',
          age: 1,
          caregiver: 'teste com nome bem grande de novo novamente outra vez',
          status: 'good',
        }} />

        <PatientCard patient={{
          name: 'teste com nome bem grande',
          age: 1,
          caregiver: 'teste com nome bem grande de novo novamente outra vez',
          status: 'caution',
        }} />

        <PatientCard patient={{
          name: 'teste com nome bem grande',
          age: 1,
          caregiver: 'teste com nome bem grande de novo novamente outra vez',
          status: 'good',
        }} />
      
      </div>
    </div>
  );
}

export default PatientList;
