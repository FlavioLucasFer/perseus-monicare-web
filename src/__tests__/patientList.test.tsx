import React from 'react';
import { render, cleanup, fireEvent, screen } from '@testing-library/react';
import { shallow } from 'enzyme';
import { Router } from 'react-router-dom';
import PatientList from 'screens/PatientList';

const mockedUseNavigate = jest.fn();

jest.mock('react-router-dom', () => ({
	...jest.requireActual('react-router-dom') as any, 
	useNavigate: () => mockedUseNavigate,
}));

describe('PatientList Screen', () => {
	it('Should render properly', () => {
		const wrapper = shallow(
			<PatientList />
		);
		expect(wrapper.debug()).toMatchSnapshot();
		
	}); 

});