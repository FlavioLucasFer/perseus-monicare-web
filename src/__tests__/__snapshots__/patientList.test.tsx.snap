// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PatientList Screen Should render properly 1`] = `
"<div className=\\"container\\" style={{...}}>
  <div className=\\"row col-sm-12\\" />
</div>"
`;
